### ISSUE TITLE
yii\mongodb\Collection::findOne() return false

### ISSUE THREAD

**Issue by AstRonin** [yiisoft/yii2-mongodb#136]
### What steps will reproduce the problem?

```php
User::getCollection()->findOne(['some_field' => 'not_find_value']);
```

### What's expected?

`array` or `null`

### What do you get instead?

`false`
Be cause function `current([]);` in method `yii\mongodb\Collection::findOne()` return `false`

### Additional info

| Q                      | A
| ---------------------- | ---
| Yii version            | 2.0.8
| Yii MongoDB version    | 2.1.0
| MongoDB server version | 3.2
| PHP version            | 5.6
| Operating system       | Win

**Comment by klimov-paul** [comment]
Resolved by commit b3f6577acccea5c49b1c7ac5ff40352afb393a54

**Comment by eramudeep** [comment]
i am still having same problem with the some of my models
